### DISCUSSION TITLE
How to contribute ?

### DISCUSSION THREAD

**Discussion by JagveerGagaan** [cake-build/cake#3797] [Extension Q&A]
Hey i am new here but i don't know to contribute in github i know HTML,CSS,JAVASCRIPT and REACTJS, If any one can can tell it'll help me but i want to mention that I already know some basic git commands.

**Comment by gep13** [comment]
@JagveerGagaan thanks for reaching out!  We are certainly happy to help with new contributions to this project.  Is there something specific that you are interested in helping with?

Cake, the build automation tool, has a number of different aspects.  From the Cake Engine, to Visual Studio and Visual Studio Code extensions, GitHub Action, documentation site, etc.

If you wanted to pick one of those, and then look through the open issues to see if there is anything you wanted to help with, and then ask more targeted questions, I am sure we can get you started with contributing to the project.

**Reply by JagveerGagaan** [reply]
@gep13 Thanks for your answer so, as I mentioned in my question I know Html, CSS, javascript, and react js if there is anything related to these I will be happy to help like website, documentation site, and other things related to these technologies.
Thanks for your time :)

**Reply by nils-a** [reply]
@JagveerGagaan, html/css/js is mostly used on the [website](https://cakebuild.net). If you'd like, you can have a look at the [open issues in cake-build/website](https://github.com/cake-build/website/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) and see if something there sounds like you want to try it.
